napari/plugins/_tests/test_plugins_manager.py

```diff
@@ -8,10 +8,7 @@
 if TYPE_CHECKING:
     from napari.plugins._plugin_manager import NapariPluginManager
 
-from napari._tests.utils import slow
 
-
-@slow(15)
 def test_plugin_discovery_is_delayed():
```

Comment:
do we want to do something here? i worry otherwise this test doesn't really test anything

Reply:
no this definitely does something without it.  the line `sys.exit(len(plugin_manager.plugins) != 2)` will return a sys.exit code that fails the test if plugins were discovered.  it's not about speed.